__tests__/index.test.js

```diff
@@ -216,3 +216,92 @@ describe("returns an error when the file is not found", () => {
       });
   });
 });
+
+describe("/api/articles/:article_id", () => {
+  test("PATCH 200: Successfully increments the votes", () => {
+    const testPatch = { inc_votes: 1 };
+    return request(app)
+      .patch("/api/articles/1")
+      .send(testPatch)
+      .expect(200)
+      .then(({ body }) => {
+        const { article } = body;
+        expect(article).toMatchObject({
+          article_id: 1,
+          title: "Living in the shadow of a great man",
+          topic: "mitch",
+          author: "butter_bridge",
+          body: "I find this existence challenging",
+          created_at: "2020-07-09T20:11:00.000Z",
+          votes: 101,
+          article_img_url:
+            "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+        });
+      });
+  });
+
+  test("PATCH 200: Successfully decrements the votes", () => {
+    const testPatch = { inc_votes: -10 };
+    return request(app)
+      .patch("/api/articles/1")
+      .send(testPatch)
+      .expect(200)
+      .then(({ body }) => {
+        const { article } = body;
+        expect(article).toMatchObject({
+          article_id: 1,
+          title: "Living in the shadow of a great man",
+          topic: "mitch",
+          author: "butter_bridge",
+          body: "I find this existence challenging",
+          created_at: "2020-07-09T20:11:00.000Z",
+          votes: 90,
+          article_img_url:
+            "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+        });
+      });
+  });
+
+  test("PATCH 400: Invalid article_id input", () => {
+    const testPatch = { inc_votes: 1 };
+    return request(app)
+      .patch("/api/articles/invalid_input")
+      .send(testPatch)
+      .expect(400)
+      .then(({ body }) => {
+        const { msg } = body;
+        expect(msg).toBe("bad request");
+      });
+  });
+
+  test("PATCH 404: article_id not found", () => {
+    const testPatch = { inc_votes: 1 };
+    return request(app)
+      .patch("/api/treasures/764837")
+      .send(testPatch)
+      .expect(404)
+      .then(({ body }) => {
+        const { msg } = body;
+        expect(msg).toBe("Not found!");
+      });
+  });
+});
```

Comment:
Fab start here. I'd also test what happens when:
- patch with an empty object {}
- patch with an object with the wrong key
- patch with a value that isn't a number